src/modules/material-requests/controllers/material-requests.controller.ts

```diff
@@ -170,7 +170,7 @@ export class MaterialRequestsController {
 			});
 
 			materialRequests.items.forEach((materialRequest: MaterialRequest) => {
-				materialRequest.contactMail = materialRequest.contactMail.find(
+				materialRequest.contactMail = materialRequest?.contactMail?.find(
```

Comment:
maybe add a comment:
```suggestion
				// If the email does not exist, the campaign monitor service will default to process.env.MEEMOO_MAINTAINER_MISSING_EMAIL_FALLBACK
				materialRequest.contactMail = materialRequest?.contactMail?.find(
```